rasa_core/policies/policy.py

```diff
@@ -4,74 +4,131 @@
 from __future__ import unicode_literals
 
 import logging
+import typing
 
-import numpy as np
+from inspect import signature
```

Comment:
@Ghostvv this is incompatible with python 2.7

Reply:
i've fixed it with using `valid_keys = getargspec(func).args` instead, as far as i know it does the same thing , it just returns a list instead of a dict